v2/payments/installing-payment-gateway.rst

```diff
@@ -0,0 +1,40 @@
+Installing and configuring a payment gateway
+=======================================
+
+While configuring your store you would probably want to install and configure a payment gateway provider module so that you can accept online payments. Currently available payment gateways for Commerce 2 are listed `here <available-payment-gateways.rst>`_.
+
+This tutorial will guide you through configuring a payment gateway provider using the Commerce PayPal module as an example.
+
+Install the Module
+------------------
+
+The first step is to install the module that provides integration with the desired payment gateway. In this case, Commerce PayPal. For information on how to install a module in Drupal 8 see `Installing Drupal 8 Modules <https://www.drupal.org/docs/8/extending-drupal-8/installing-drupal-8-modules>`_ and `Installing Modules from the Command Line <https://www.drupal.org/docs/8/extending-drupal-8/installing-modules-from-the-command-line>`_.
```

Comment:
I didn't make the comment if it wasn't really dependency. Many payment gateway use an SDK and require Composer. PayPal does not due to the license of their SDK. However Braintree, Authnet and others use third party libraries